packages/vscode-extension/src/commands/ImportObjects.ts

```diff
@@ -0,0 +1,238 @@
+			return;
+		}
+
+		const scriptId = await this.promptSelectedScriptId();
+
+		if (scriptId == undefined) {
+			this.messageService.showInformationMessage(this.translationService.getMessage(IMPORT_OBJECTS.PROCESS_CANCELED));
+			return;
+		}
+
+		const listObjectsResult = await this.listObjects(appId, selectedObjectTypes, scriptId);
+		if (listObjectsResult.status !== 'SUCCESS' || !listObjectsResult.data || listObjectsResult.data.length == 0) {
+			this.showOutput(listObjectsResult);
+			return;
+		}
+
+		const selectedScriptIds = await this.promptObjects(listObjectsResult.data);
+
+		if (!selectedScriptIds || selectedScriptIds.length == 0) {
+			this.messageService.showCommandError(this.translationService.getMessage(IMPORT_OBJECTS.ERROR.EMPTY_LIST));
+			return;
+		}
+		const includeReferencedFiles = await this.promptIncludeReferencedFiles();
+		const overwrite = await this.promptOverwrite(includeReferencedFiles);
+		if (!overwrite || overwrite === this.translationService.getMessage(ANSWERS.NO)) {
+			this.messageService.showInformationMessage(this.translationService.getMessage(IMPORT_OBJECTS.PROCESS_CANCELED));
+			return;
+		}
+
+		const actionResult = await this.customObjectService.importObjects(
+			destinationFolder,
+			appId,
+			selectedScriptIds,
+			includeReferencedFiles === this.translationService.getMessage(ANSWERS.YES),
+			this.executionPath
+		);
+
+		this.showOutput(actionResult);
+		return actionResult;
+	}
+
+	private getDestinationFolder(pathDir: string) {
+		const isDirectory = fs.lstatSync(pathDir).isDirectory();
+		const directoryName = isDirectory ? pathDir : path.dirname(pathDir);
+		const destinationFolder = directoryName.split(this.getProjectFolderPath())[1].replace(/\\/gi, '/');
+		return destinationFolder;
+	}
+
+	private async promptAppId(projectInfoService: { getApplicationId: () => string }) {
+		const filterAppId = await window.showQuickPick(
+			[this.translationService.getMessage(ANSWERS.YES), this.translationService.getMessage(ANSWERS.NO)],
+			{
+				placeHolder: this.translationService.getMessage(IMPORT_OBJECTS.QUESTIONS.FILTER_APP_ID),
+				canPickMany: false,
+			}
+		);
+
+		if (!filterAppId) {
+			return;
+		}
+
+		if (filterAppId && filterAppId == this.translationService.getMessage(ANSWERS.NO)) {
+			return '';
+		}
+
+		const defaultAppId = projectInfoService.getApplicationId();
+		let appId = await window.showInputBox({
+			ignoreFocusOut: true,
+			placeHolder: this.translationService.getMessage(IMPORT_OBJECTS.QUESTIONS.APP_ID, defaultAppId),
+			validateInput: (fieldValue) => {
+				if (fieldValue === '') {
+					return null; //In this case empty field is valid
+				}
+				let validationResult = InteractiveAnswersValidator.showValidationResults(fieldValue, InteractiveAnswersValidator.validateSuiteApp);
+				return typeof validationResult === 'string' ? validationResult : null;
+			},
+		});
+		if (appId == undefined) {
```

Comment:
We should use triple equal signs here (===) instead of just two equal signs (==).